app/templates/app/delete_installment.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html' %}
+{% load bootstrap4 %}
+{% bootstrap_css %}
+{% load static %}
+{% load staticfiles %}
+{% load i18n %}
+{% load render_table from django_tables2 %}
+{% block content %}
+<html>
+    <body>
+    <div class='m-5 p-3'>
+        <form method="post">{% csrf_token %}
+        <div class="row justify-content-center m-3">
+                    <div class="card" style="width: 30rem;">
+                      <div class="card-body">
+                        <h5 class="card-title">Delete Installment</h5>
+                        <p class="card-text center">
+                          <li class="m-2"> <b>Organizer: </b> {{ object.contract.organizer_account_name }}</li>
+                          <li class="m-2"> <b>Recoupable: </b> {{ object.is_recoup }}</li>
+                          <li class="m-2"> <b>Max payment date: </b> {{ object.maximum_payment_date }}</li>
+                          <li class="m-2"> <b>Upfront projection: </b> {{ object.upfront_projection }}</li>
+                          <li class="m-2"> <b>Recoup amount: </b> {{ object.recoup_amount }}</li>
+                          <li class="m-2"> <b>Status: </b> {{ object.status }}</li>
+                          <li class="m-2"> <b>GTF: </b> {{ object.gtf }}</li>
+                          <li class="m-2"> <b>GTS: </b> {{ object.gts }}</li>
+                          <div class="row m-4">
+                              <div class="col justify-content-md-center">
+                                  <a href="{% url 'installments-create' object.contract_id %}" role="button" class="btn btn-outline-secondary"><i class="far fa-arrow-alt-circle-left"></i>  Back</a>
+                              </div>
+                              <form method="post">{% csrf_token %}
```

Comment:
Why you need this `<form>` , what's about of line 12?

Reply:
I don't need it, I made a mistake.